lib/bolt/target.rb

```diff
@@ -258,6 +282,22 @@ def to_h
       )
     end
 
+    def detail
+      {
+        'name' => name,
+        'uri' => uri,
```

Comment:
wont name always be the same as uri for Target? I think we should only print uri

Reply:
No, you can set name and URI separately:
```
targets:
  - uri: dhix35pjb8hiya9.delivery.puppetlabs.net
    name: myhost
```

`bolt command run hostname -n myhost`

Reply:
This is for inventory v1 target v1. I kept the URI for v2.